src/modules/ProductDetailsPage/About/About.tsx

```diff
@@ -1,5 +1,28 @@
-// import styles from './About.module.scss';
+import React from 'react';
+import styles from './About.module.scss';
+import { PhoneDescription } from '@/types/Product';
 
-export const About = () => {
-  return <></>;
+interface AboutProps {
+  description: PhoneDescription[];
+}
+
+export const About: React.FC<AboutProps> = ({ description }) => {
+  return (
+    <section className={styles.about}>
+      <h2 className={styles.about__title}>About</h2>
+      {description.map((item, index) => (
+        <div key={index} className={styles.about__text_container}>
```

Comment:
Please, split the code into separate components. All the code after `description.map((item, index) => (`  in one component and after `item.text.map((paragraph, idx) => (` in another.